docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst

```diff
@@ -385,29 +450,98 @@ Input Data Table
 Validation Criteria
 +++++++++++++++++++
 
-.. todo::
+ - Model 1 (Susceptible to MM): compare simulation baseline results of MM prevalence, 
+   MM incidence, and MM cause-specific mortality stratified by age, sex, and year to 
+   GBD 2019 age-/sex-specific MM estimates.
+ - Model 2 (MM to MM_{Nth}_relapse): compare simulation baseline results of overall 
+   survival and progression-free survival by disease state to line-specific survival 
+   outcomes obtained from [Braunlin-et-al-2020]_.
+
+.. list-table:: Count measures from simulation stratified by disease state and time
+   :widths: 1 10
+   :header-rows: 1
+   
+   * - Measure
+     - Definition
+   * - disease_state
+     - indication of health status 
+   * - t_start
+     - start time since entrance of simulantion (months)
```

Comment:
since entrance of *disease_state*, I think

Reply:
You are right, we care about the time simulants spent in a specific disease state, will update this definition.